instance/management/commands/migrate_ephemeral.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2016 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+    This management command will migrate instances using ephemeral databases
+    to external databases instead.
+
+    This is intended to deprecate ephemeral databases.
+"""
+
+import logging
+
+from django.core.management.base import BaseCommand
+
+from instance.models.openedx_instance import OpenEdXInstance
+
+LOG = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    migrate_ephemeral management command class
+    """
+    help = (
+        'Disables `use_ephemeral_databases` on all instances and provisions AppServers when required.'
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(Command, self).__init__(*args, **kwargs)
+        self.options = {}
+        self.retried = {}
+
+    def handle(self, *args, **options):
+        """
+        Find instances using ephemeral databases and migrate to external ones
```

Comment:
@viadanna Nit: This is a little terse. Maybe just repeat the contents of the `help` text above in this docstring to beef it up a little:

```
        Disable ephemeral databases for all instances that are currently using them
        and provision new app servers pointing to persistent databases.
```